### PR TITLE
Deletion: improve logging a bit

### PR DESCRIPTION
**Pull request by aermakov-zalando**
This would make them a bit nicer to read.

### CODE REVIEW

File: provisioner/clusterpy.go [hunk 1]
```diff
@@ -1031,17 +1031,17 @@ func (p *clusterpyProvisioner) Deletions(logger *log.Entry, cluster *api.Cluster
...
-		logger.Infof("Skipping deletion of %s: resource not found", name)
+		logger.Infof("Skipping deletion of %s %s: resource not found", kind, name)
 		return nil
 	}
 	if err != nil {
 		return fmt.Errorf("unable to delete: %w", err)
 	}
 
-	logger.Infof("Resource %s deleted", name)
+	logger.Infof("%s %s deleted", name)
```
Comment (muaazsaleem) [inline provisioner/clusterpy.go:1044]: Did we forget `kind` here?

Comment (aermakov-zalando) [reply provisioner/clusterpy.go:1044]: Oops.

**Comment by muaazsaleem** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**